app/controllers/users_controller.rb

```diff
@@ -1,7 +1,50 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = [
+      {
+        name: 'Giuseppe',
+        photo: 'https://images-na.ssl-images-amazon.com/images/I/41vMBIeF-zL._SX384_BO1,204,203,200_.jpg',
+        bio: 'Did my opinion offend you? You should hear the ones I don’t say out loud.',
+        postsCounter: 2
+      },
+      {
+        name: 'Marco',
+        photo: 'https://images-na.ssl-images-amazon.com/images/I/41vMBIeF-zL._SX384_BO1,204,203,200_.jpg',
+        bio: 'A bus station is where a bus stops. A train station is where a train stops.',
+        postsCounter: 1
+      },
+      {
+        name: 'Simon',
+        photo: 'https://images-na.ssl-images-amazon.com/images/I/41vMBIeF-zL._SX384_BO1,204,203,200_.jpg',
+        bio: 'Born at a very young age.',
+        postsCounter: 2
+      }
+    ]
+  end
```

Comment:
This is bad practice, you're expected to have this data in your DB so the command `@users = User.all` renders all the users from the DB to the action view. You create seed data for app testing.

- [ ] Please update code